src/Microsoft.Azure.SignalR.Management/ServiceHubContext.cs

```diff
@@ -20,6 +20,12 @@ public abstract class ServiceHubContext : IServiceHubContext, IDisposable
         public virtual UserGroupManager UserGroups => null;
         IUserGroupManager IServiceHubContext.UserGroups => UserGroups;
 
+        /// <summary>
+        /// Gets a streaming manager instance which implements <see cref="IStreamingManager"/> that can be used to send streaming messages to clients.
+        /// </summary>
+        public virtual StreamingManager Streaming => null;
```

Comment:
We implement the `IServiceHubContext` interface here just for backward compatibility. For new features, I don't think we need to add it to `IServiceHubContext` anymore.

Reply:
so, i can remove it from `IServiceHubContext`, right?

Reply:
Yes